src/libs/brand-utils.js

```diff
@@ -1,38 +1,35 @@
-import { brands } from 'src/libs/brands'
+import _ from 'lodash/fp'
+import { brands, defaultBrand } from 'src/libs/brands'
 import { getConfig } from 'src/libs/config'
-import * as Utils from 'src/libs/utils'
 
 
 export const isBrand = brand => {
-  return new RegExp(`^((dev|alpha|perf|staging)\\.)?${brand.hostName}$`).test(window.location.hostname)
+  return new RegExp(`^((dev|alpha|staging)\\.)?${brand.hostName}$`).test(window.location.hostname)
```

Comment:
Not directly related to the ticket. Opportunistically removing `perf` env reference that was left over.